composer.json

```diff
@@ -11,20 +11,21 @@
         {"name": "Jonathan H. Wage", "email": "jonwage@gmail.com"}
     ],
     "require": {
-        "php": ">=5.3.2",
+        "php": "^5.5",
         "doctrine/mongodb-odm": "~1.0",
         "psr/log": "~1.0",
-        "symfony/options-resolver": "~2.1",
-        "symfony/doctrine-bridge": "~2.1",
-        "symfony/framework-bundle": "~2.1",
-        "symfony/dependency-injection": "~2.2",
-        "symfony/console": "~2.1",
-        "symfony/config": "~2.2"
+        "symfony/options-resolver": "~2.8|~3.0",
```

Comment:
Maybe it's time to follow Composer new conventions.

`~2.8` -> `^2.8`.

`^2.8|^3.0` -> `^2.8 || ^3.0`